client/src/hooks/UseFetch.ts

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect, useCallback } from "react";
+import axios, { Method } from "axios";
+
+interface FetchDataParams {
+  url: string;
+  method: Method;
+  headers: Record<string, unknown>;
+  body: Record<string, unknown>;
+}
+
+axios.defaults.baseURL = "http://localhost:3000";
```

Comment:
we are not going to use localhost. You can enter the address of your api. can we set the GET method by default?

Reply:
Sure, I've just changed it